### ISSUE TITLE
Error loading discount code

### ISSUE THREAD

**Issue by raulrpearson** [gatsbyjs/store.gatsbyjs.org#259]
When accessing store.gatsbyjs.org and logging in with my GitHub account, I get the following error on the sidebar:

> There was an error loading your discount code.
> Here’s what came back from the server:
> Error: GraphQL error: Cannot read property 'map' of undefined
> Please reload the page and try again. If a page refresh doesn’t clear things up, please open an issue and we’ll figure out what’s going on.

Tried on macOS 10.14.3 and Firefox 65.0.1 and Chrome Version 73.0.3683.75 (Official Build) (64-bit).

This might or might not be related to some change to an email field that customer service had to make somewhere to solve some problems I was having with applying the discount code at the end of the check out process.

**Comment by jlengstorf** [comment]
@raulrpearson are you still seeing this error? I can’t seem to reproduce it — I’m wondering if we were having server trouble or a temporary upstream outage.

Let me know when you get a chance. Thanks!

**Comment by raulrpearson** [comment]
@jlengstorf, I got my swag delivered, which means my Level 1 Swag Code has been claimed, which I think means that I can't reproduce the error, since the state of my profile has changed.

I believe there was some manual processing done to get my order through. I think it involved someone manually changing my email address somewhere, which might have caused the hiccup.

I tried login in just now and I notice the flow an interface has changed slightly since I reported this issue. Everything seems to be working fine.

I'm okay with having this issue closed. I can reopen or open a new one if I have any problems when I try claiming my Level 2 Swag Code.

Thanks for looking into this!
